### ISSUE TITLE
[BUG] [Formatter] django blocktranslate wrong indentation

### ISSUE THREAD

**Issue by vasekch** [Riverside-Healthcare/djLint#493]
possible regression of #26 

`djlint, version 1.19.7`

## Issue

Something is wrong with blocktranslate formatting, `{% endblocktranslate %}` is not properly indented.

## Code example - desired state

```html
<div>
    <span>
        {% translate "View all" %}
        <br />
        ({% blocktranslate count counter=images|length trimmed %}
            {{ counter }} photo
            {% plural %}
            {{ counter }} photos
        {% endblocktranslate %})
        <br/>
        <span>remark</span>
    </span>
</div>
```

## Formatter output

```html
<div>
    <span>
        {% translate "View all" %}
        <br />
        ({% blocktranslate count counter=images|length trimmed %}
            {{ counter }} photo
            {% plural %}
            {{ counter }} photos
{% endblocktranslate %})
        <br/>
        <span>remark</span>
    </span>
</div>
```

The example uses complex version of blocktranslate, but result is the same even if I remove `count`/`trimmed` args. Also wrapping content brackets `(`, `)` ~~don't seem to play any role in this issue~~. *see the comment*

Current workaround is to wrap all code in `djlint:off`.

Thank you for djLint :green_heart:!

---

**Comment by vasekch** [comment]
Ha! My bad.

It *is* the closing bracket after the end tag `{% endblocktranslate %})`. If removed all works as it should.

I got confused by the fact that I need to put all spaces in front of endblocktranslate manually as they are part of not formatted content.

I guess in my case the most elegant solution is to put brackets into the translation itself and let blocktranslate start/end tags on separate lines.

I'm not sure if having any content after end tag is in favor of djLint formatting strategy. Feel free to close.

**Comment by vasekch** [comment]
Putting simple space at the end of the line with `{% endblocktranslate %} ` also breaks the indent after formatting, so I think it is the issue.

**Comment by christopherpickering** [comment]
Hmm, good issue. We can probably get it fixed. In v2, I think will not be an issue, but for now we can probably fix it up.

**Comment by christopherpickering** [comment]
@vasekch sorry for the delay on this :) I'm going to release an update that allows closing `{% endblocktrans...%}` tags to be indented if they have a leading space.

**Comment by christopherpickering** [comment]
commit d6672734 fixes this issue.
